virttest/utils_sriov.py

```diff
@@ -77,6 +77,25 @@ def set_vf_mac(ethname, mac_addr, vf_idx=0, session=None):
             cmd, shell=True, verbose=True, session=session)
 
 
+def get_vf_mac(ethname, vf_idx=0, session=None):
+    """
+    Get mac address for VF
```

Comment:
Suggest to add more information here, it may make others confused as vf mac can be the mac we set in the vm, or the output in "ifconfig $vf"(they may be different in some cass). In this function, we get the VF's original admin mac.

Reply:
Thank you @yalzhang for your comment! I think we should not limit this function to get the VF's **original** admin mac. It can also use to get the mac after setting it to other address.  I did a minor update in docstring, could you please re-review?

Reply:
You are right. It looks better for me now! :-)